docs/reference/composition.md

```diff
@@ -637,6 +637,13 @@ Currently only `multiply` is supported.
      type: Convert
      convert: FromBase64
 
+# If the value of the 'from' field is not nil, the value of the 'to' field will be
```

Comment:
@plumbis This PR has this doc change. Having this updated here is enough, or do we need a separate PR in the new [docs repo](https://github.com/crossplane/docs)?